index.html

```diff
@@ -0,0 +1,189 @@
+          </ul>
+        </nav>
+        <button class="campaign">Campaign</button>
+      </div>
+    </header>
+    <main>
+      <section class="intro">
+        <div class="menu">
+          <i class="fas fa-bars fa-2x"></i>
+        </div>
+        <h2>"LET U ALL HELP SAVE OUR CLIMATE"</h2>
+        <h1>GLOBAL CLIMATE FIXING CONFERENCE 2022</h1>
+        <div class="intro-descr">
+          <p>
+            This is the official page of  the World Climate Fixing Conference
+            which is held annually. This conference is held tech company that 
+            are solving the world problems of climate condition and change.
+          </p>
+        </div>
+        <div class="date-local">
+          <h3>2022.03.016(WED) ~ 11(FRI)</h3>
+          <h4>Silicon Valley, Carlifornia, U.S.A</h4>
+        </div>
+      </section>
+      <section class="main-program" id="program">
+        <h2>Main Program</h2>
+        <hr />
+        <div class="programs">
+          <div class="program">
+            <img
+              class="program-icon"
+              src="./images/lecturers.png"
+              alt="lecture"
+            />
+            <h3 class="program-title">Lecture</h3>
+            <p class="program-descr">
+              Listen to the expert from various diverse locations on climate solutions.
+            </p>
+          </div>
+          <div class="program">
+            <img
+              class="program-icon"
+              src="./images/visitors.png"
+              alt="Exhibition"
+            />
+            <h3 class="program-title">Exhibition</h3>
+            <p class="program-descr">
+              Appreciate talent into this field that are solving the world problems.
+            </p>
+          </div>
+          <div class="program">
+            <img class="program-icon" src="./images/chat.png" alt="Forum" />
+            <h3 class="program-title">Forum</h3>
+            <p class="program-descr">
+              Listen to the expert from various diverse locations on climate solutions.
+            </p>
+          </div>
+          <div class="program">
+            <img
+              class="program-icon"
+              src="./images/workshop.png"
+              alt="workshops"
+            />
+            <h3 class="program-title">Workshop</h3>
+            <p class="program-descr">
+              Share your thoughts and opinions with top experts in the field of technology.
+            </p>
+          </div>
+          <div class="program">
+            <img
+              class="program-icon"
+              src="./images/global-network.png"
+              alt="ignite"
+            />
+            <h3 class="program-title">Ignite</h3>
+            <p class="program-descr">
+              Get a life time opportuniy to network with  experts and show case your 
+              ideas to fellow technocrat.
+          </p>
+          </div>
+        </div>
+        <a class="program-link" href="/">SEE THE WHOLE PROGRAM</a>
```

Comment:
- [ ] Kindly set the style of this link to `text-decoration: underline`

Reply:
@Cathella . Thanks for your feedback.
Kindly review changes made to the README file and also loom video

Reply:
- [ ] the requested changes are not implemented yet. Kindly work on making the changes requested by the previous reviewer.